build/setup.d/95-nvidia.sh

```diff
@@ -3,7 +3,7 @@
 # See https://github.com/NVIDIA/nvidia-docker/wiki/Deploy-on-Amazon-EC2
 # But we're using the run script to not depend on build tools
 DRIVER_ARCH="Linux-x86_64"
-DRIVER_VERSION="375.26"
+DRIVER_VERSION="384.81"
```

Comment:
http://us.download.nvidia.com/XFree86/Linux-x86_64/384.81/NVIDIA-Linux-x86_64-384.81.run leads to a 404, are you sure the version number is correct?

Reply:
hmmm, ya you're right. I just found the version number here http://www.nvidia.com/download/driverResults.aspx/124728/en-us the download link appears to be different

Reply:
it appears from http://www.nvidia.com/object/unix.html that 384.90 is the latest version that supports to same cards as 375.26.

Oddly, the product page seems to say that neither supports Tesla cards, despite having used 375.26 with tesla K80s.

*TLDR* I think it should be 384.90, I will update